server/internal/server/org.go

```diff
@@ -0,0 +1,75 @@
+package server
+
+import (
+	"context"
+
+	v1 "github.com/llm-operator/user-manager/api/v1"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// CreateOrganization creates a new organization.
+func (s *S) CreateOrganization(ctx context.Context, req *v1.CreateOrganizationRequest) (*v1.Organization, error) {
+	if req.Title == "" {
+		return nil, status.Error(codes.InvalidArgument, "title is required")
+	}
+
+	orgID := ""
```

Comment:
Is this intentional? My understanding is that we need to assign an org ID here.

Reply:
Ah, thanks. I forgot to assign it.